src/WebApi/Rsp.IrasService.WebApi/Controllers/ReviewBodyController.cs

```diff
@@ -119,9 +119,37 @@ public async Task<ReviewBodyAuditTrailResponse> GetAuditTrailForReviewBody(Guid
         return result;
     }
 
+    /// <summary>
+    ///     Enable a review body
+    /// </summary>
+    /// <param name="reviewBodyDto">Research Body Dto</param>
+    [HttpPut("enable/{id}")]
+    public async Task<ReviewBodyDto?> Enable(Guid id)
+    {
+        var request = new EnableReviewBodyCommand(id);
+        var enableReviewBodyResult = await mediator.Send(request);
+
+        // log audit trail
+        var userId = UserEmail(User);
+        if (enableReviewBodyResult != null)
+        {
+            var auditRecord = auditService.GenerateAuditTrailDtoFromReviewBody(
+                enableReviewBodyResult,
+                userId!,
+                ReviewBodyAuditTrailActions.Enable
+            );
+
+            var loggedAuditTrail = await auditService.LogRecords(auditRecord);
+        }
+
+        return enableReviewBodyResult;
+    }
+
     // gets the currently logged in user's email from the user claims
     private static string? UserEmail(ClaimsPrincipal user)
     {
-        return user?.Claims?.FirstOrDefault(x => x.Type == "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress")?.Value;
+        return user?.Claims
+            ?.FirstOrDefault(x => x.Type == "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress")
```

Comment:
You don't need to hardcode this. There is a built-in enum for checking claim types.

```suggestion
            ?.FirstOrDefault(x => x.Type == ClaimTypes.Email)
```

I would also suggest creating an extension method on the controller for User, so that it can be used in other controllers if needed